Makefile

```diff
@@ -44,13 +44,13 @@ build-%:
 
 .PHONY: docker-build-push
 docker-build-push: docker-build-prepare ## Build and push a cross-arch Docker image tagged with DOCKER_IMAGE_NAME.
-	$(MAKE) build-x86_64-unknown-linux-gnu
```

Comment:
related, if we do add `gcp` we should add the flag here as well: https://github.com/foundry-rs/foundry/pull/8232

Reply:
yep, we have https://github.com/foundry-rs/foundry/issues/10221 to track gcp